taker/src/main.rs

```diff
@@ -340,6 +341,13 @@ async fn main() -> Result<()> {
         taker_peer_id: identities.libp2p.public().to_peer_id().to_string(),
     };
 
+    let environment = match option_env!("ITCHYSATS_ENVIRONMENT") {
+        Some(environment) => environment
+            .parse()
+            .expect("String that is not processable to default to unknown environment"),
```

Comment:
🔧 It would be safer if you don't use the fallible method `parse`. Instead of implementing `FromStr` we could introduce a custom method that takes an `&str` and is infallible.